### ISSUE TITLE
nuscenes_infos_temporal_test.pkl没有使用

### ISSUE THREAD

**Issue by Wjbbbbb** [OpenDriveLab/UniAD#241]
projects/configs/stage2_e2e/base_e2e.py, line 481:
为什么ann_file_test还是nuscenes_infos_temporal_**val**.pkl，不是nuscenes_infos_temporal_**test**.pk？
？
ann_file_train=info_root + f"nuscenes_infos_temporal_train.pkl"
ann_file_val=info_root + f"nuscenes_infos_temporal_val.pkl"
ann_file_test=info_root + f"nuscenes_infos_temporal_val.pkl"

**Comment by sephyli** [comment]
That is correct. In the UniAD project, evaluation is typically only performed on the NuScenes validation set (nuscenes_infos_temporal_val.pkl).

The reason is that the NuScenes test set is reserved for tasks like detection, and we cannot access the necessary ground-truth labels (for detection, prediction, occupancy, planning collision, etc.) required for our evaluation.

If your goal is to run inference on the test set, you will need to manually modify the corresponding config file. Be aware that you may also need to adjust some code in dataset/dataloader as it relies on reading certain labels.
